### PR TITLE
Uncomment config :opentelemetry, :resource, settings.

### PR DESCRIPTION
**Pull request by jinjagit**
Try using these config settings.

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -43,8 +43,8 @@ config :opentelemetry,
...
 # OTEL_RESOURCE_ATTRIBUTES=service.name=name,service.namespace=namespace
 
-# config :opentelemetry, :resource,
-#   service: [
-#     name: "service-name",
-#     namespace: "service-namespace"
-# ]
+config :opentelemetry, :resource,
+  service: [
+    name: "service-name",
```
Comment (oliverbarnes) [inline config/config.exs:48]: would be good to now what `resource`, `service` and `namespace` all refer to here. `service-name` and `service-namespace` look like placeholder values

**Review by oliverbarnes — CHANGES_REQUESTED** [review]

File: config/config.exs (hunk 1 above)
Comment (jinjagit) [reply config/config.exs:48]: Agree. I have done some searching on that, but to no avail as yet.

Comment (jinjagit) [reply config/config.exs]: Let's try without these (commented out again), but with dummy data?

Comment (oliverbarnes) [reply config/config.exs]: worth a try 👍

**Review by oliverbarnes — APPROVED** [review]

**Pull request merged**